src/CircuitZoo/CircuitZoo.jl

```diff
@@ -37,6 +37,23 @@ struct Purify2to1 <: AbstractCircuit
     end
 end
 
+struct Purify3to1 <: AbstractCircuit
+    leaveout::Symbol
```

Comment:
May we remove this for now, until the correct set of parameters is derived. For the moment it can be just an empty struct without any parameters